llvm/test/tools/yaml2obj/GOFF/end-only.yaml

```diff
@@ -0,0 +1,22 @@
+# RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
+
+## Verify that the GOFF end record is correct, with all fields having a value.
```

Comment:
Can you change this to state that we're verifying that all fields have the value we expect but that the GOFF is still invalid? We have a lot of developers who don't know GOFF and stating that the record "is correct" won't be helpful to them.